compiler/packages/react-compiler-healthcheck/src/checks/reactCompiler.ts

```diff
@@ -121,5 +121,36 @@ export default {
         `Successfully compiled ${SucessfulCompilation.length} out of ${totalComponents} components.`
       )
     );
+
+    if (verbose) {
+      for (const compilation of [...SucessfulCompilation, ...ActionableFailures, ...OtherFailures]) {
+        const filename = compilation.fnLoc?.filename;
+
+        if (compilation.kind === "CompileSuccess") {
+          const name = compilation.fnName;
+          const isHook = name?.startsWith('use');
+
+          if (name) {
+            console.log(
+              chalk.green(
+                `Successfully compiled ${isHook ? "hook" : "component" } [${name}](${filename})`
+              )
+            );
+          } else {
+            console.log(chalk.green(`Successfully compiled ${compilation.fnLoc?.filename}`));
+          }
+        }
+
+        if (compilation.kind === "CompileError") {
+          const reason = compilation.detail.description;
```

Comment:
I found description to be null in most cases. For example:

```
{
  kind: 'CompileError',
  fnLoc: {
    start: { line: 34, column: 15, index: 993 },
    end: { line: 68, column: 1, index: 1956 },
    filename: 'src/components/StationFinder/components/StationsMarkers.tsx',
    identifierName: undefined
  },
  detail: {
    reason: 'React Compiler has skipped optimizing this component because the existing manual memoization could not be preserved. This value may be mutated later, which could cause the value to change unexpectedly',
    description: null,
    severity: 'CannotPreserveMemoization',
    suggestions: null,
    loc: {
      start: [Object],
      end: [Object],
      filename: 'src/components/StationFinder/components/StationsMarkers.tsx',
      identifierName: undefined
    }
  }
}
```

Also, adding exact error location would be nice.

So something like this could work better:

```ts
        if (compilation.kind === "CompileError") {
          const { reason, loc } = compilation.detail;

          const lnNo = loc.start?.line;
          const colNo = loc.start?.column;

          console.log(
            chalk.red(
              `Failed to compile ${
                filename
              }${
                lnNo !== undefined ? `:${lnNo}${
                  colNo !== undefined ? `:${colNo}` : ""
                }` : ""
              }${
                reason? `\n Reason: ${reason}` : ""
              }`
            )
          );
        }
```